website/static/js/projectSettings.js

```diff
@@ -67,19 +101,45 @@ var NodeCategorySettings = oop.extend(
                 .then(function(response) {
                     return response.updated_fields.category;
                 })
-                .done(self.updateSuccess.bind(self))
-                .fail(self.updateError.bind(self));
+                .done(self.updateCategorySuccess.bind(self))
+                .fail(self.updateCategoryError.bind(self));
+        },        
+        updateTitle: function() {
+            var self = this;
+            return $osf.putJSON(self.titleDescriptionEditUrl, {
+                    title: $osf.htmlEscape(self.title()),
+                })
+                .done(self.updateDescription.bind(self))
```

Comment:
@lyndsysimon Right ok. After Steve clarified that we should use one button to save changes to both title and description, I originally I had one update handler that made two ajax requests but it didn't work properly. Therefore, I made it so that the title update handler gets called first, and if the request is successful, it then goes to the description update handler which has its own ajax request. If that completes without error as well, there's one corresponding title/description success handler which only reloads the window now, but that I will change to output a success message.